most_executed_script.gs

```diff
@@ -0,0 +1,166 @@
+/**
+* Get the maximum number of executions of a cloud project
+* @param {string} cloudProjectId of the cloud project
+* @param {Date} fromTime is the date and time 
+* from which the executions has to be seen
+* @return {Object} having key as projectId and value as number of executions
+*/
+function getNumberOfExecutionOfScript(cloudProjectId, fromTime) {
+  var pageToken = null;
+  var resultData = null;
+  var max = 0;
+  var i;
+  var limit = false;
+  var processIdsMap = {};
+  var projectIdsMap = {};
+  // loop to get the first page which has enteries in it
+  do {
+    var header = {
+      'Authorization': 'Bearer ' + ScriptApp.getOAuthToken()
+    };
+    var body = {
+      'projectIds': [
+        cloudProjectId
+      ],
+      'resourceNames': [
+        'projects/' + cloudProjectId
+      ],
+      'pageToken': pageToken,
+      'orderBy': "timestamp desc"
+    };
+    var options = {
+      'method': 'post',
+      'contentType': 'application/json',
+      'headers': header,
+      'payload': JSON.stringify(body),
+      'muteHttpExceptions': false
+    };
+    var url = 'https://logging.googleapis.com/v2/entries:list';
+    var response = UrlFetchApp.fetch(url, options);
+    var json = response.getContentText();
+    resultData = JSON.parse(json);
```

Comment:
this is overwritten in each iteration. is this intentional?

Reply:
Yes
Each time page Token is changed so option needs to be updated
Other things remain the same